app/views/layouts/application.html.erb

```diff
@@ -41,35 +41,11 @@
           </ul>
         </div>
       </nav>
+    </div>
 
-      <!-- Content begin -->
-      <div class="container">
-        <div class="row">
-          <div class="col l11">
-            <%= yield %>
-          </div>
-          <div class="col l1">
-            <!-- <p>Hate popups? Refer a friend and we'll disable them for you.</p> -->
-            <div class="side-banner">
-              <!-- AWEmpire 1 -->
-              <div data-awe-container-id="791140" style="display:inline-block;width:160px;height:600px;"></div><script src="//awept.com/embed/vb?c=791140&categoryName=girl&pageName=listpage&color=brown_yellow&psid=tightfitcams&pstool=316_2&siteId=jasmin&cobrandId=&psprogram=pps&campaign_id=&subAffId={SUBAFFID}"></script>
-              <!-- AWEmpire 2 -->
-              <a title="Live webcam sex! More than 20000 Hot Girls are waiting for you!" href="//awejmp.com/?siteId=jasmin&cobrandId=&superCategoryName=[--SUPERCATEGORY--]&categoryName=girl&pageName=listpage&prm[psid]=tightfitcams&prm[pstool]=201_2&prm[psprogram]=pps&prm[campaign_id]=&subAffId={SUBAFFID}" align="center" target="_blank">
-                <img class="img" alt="Live webcam sex! More than 20000 Hot Girls are waiting for you!" src="//static1.awestatic.com/npt/banner/s1_girl/160xx600.jpg?sid=9300fc3b&psid=tightfitcams&pstool=201_2&psprogram=pps&campaign_id=" border="0" />
-              </a>
-
-              <!-- Exoclick Game Ad -->
-              <script type="text/javascript">var ad_idzone = "2808188", ad_width = "160", ad_height = "600";</script>
-              <script type="text/javascript" src="https://ads.exosrv.com/ads.js"></script>
-              <noscript><a href="https://main.exosrv.com/img-click.php?idzone=2808188" target="_blank"><img src="https://syndication.exosrv.com/ads-iframe-display.php?idzone=2808188&output=img&type=160x600" width="160" height="600"></a></noscript>
-              <!-- Exoclick Popunder -->
-              <script type="text/javascript">var ad_idzone = "2822626", ad_frequency_period = 720, ad_frequency_count = 1;</script>
-              <script type="text/javascript" src="https://ads.exosrv.com/popunder1000.js"></script>
```

Comment:
wait we need this though

Reply:
totally my bad. adding back in all of them